easybuild/easyconfigs/c/CUDAcompat/CUDAcompat-11.6.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'ModuleRC'
+
+name = 'CUDAcompat'
+version = '11.6'
+
+homepage = 'https://docs.nvidia.com/deploy/cuda-compatibility/index.html'
+description = """Using the CUDA Forward Compatibility package,
+ system administrators can run applications built using a newer toolkit
+ even when an older driver that does not satisfy the minimum required driver version
+ is installed on the system.
+ This forward compatibility allows the CUDA deployments in data centers and enterprises
+ to benefit from the faster release cadence and the latest features and performance of CUDA Toolkit.
+"""
+
+toolchain = SYSTEM
+
+dependencies = [(name, version, '-515.48.07')]
```

Comment:
I think you have a mix up here, `CUDAcompat-11.7.eb` has not been updated and the driver version here should be `510.73.08`

Reply:
Yep, C&P issue in the 11.7 file and EB helpfully changed the filename. Reuploading and also removing the hidden flag